### PR TITLE
Fix typos and misc cleanup.

### PR DESCRIPTION
**Pull request by maflcko**
Should fix https://github.com/bitcoin/bitcoin/pull/7147#issuecomment-161279446

### CODE REVIEW

**Comment by MarcoFalke** [comment]
Includes some easy cherry-picks from theuni/trivial. @theuni Are you going to do the actual rebase?

**Comment by laanwj** [comment]
No translation changes, looks sane to me
ACK

File: contrib/seeds/generate-seeds.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/python
-# Copyright (c) 2014 Wladmir J. van der Laan
+# Copyright (c) 2014 Wladimir J. van der Laan
```
Comment (laanwj) [inline contrib/seeds/generate-seeds.py]: Wladmir :-)

Comment (sipa) [inline contrib/seeds/generate-seeds.py]: Twin brother?

**Comment by laanwj** [comment]
May be nice to fix this one (from #6216) in the same go: https://github.com/petertodd/bitcoin/commit/db6047d61b742be07442f891e70570b791c585e3#commitcomment-14733536

**Comment by MarcoFalke** [comment]
Rebased and fixed nit

**Pull request merged**